boutiques_descriptors/deepprep_24_1_2.json

```diff
@@ -44,7 +44,129 @@
             "name": "fs_license_file",
             "optional": false,
             "type": "File",
-            "value-key": "<fs_license_file>"
+            "value-key": "[fs_license_file]"
+        },
+        {
+            "description": "A space delimited list of participant identifiers or a single identifier (the sub- prefix can be removed)",
+            "id": "participant_label",
+            "name": "participant_label",
+            "optional": true,
+            "type": "String",
+            "value-key": "[--participant_label '001 002 003']"
+        },
```

Comment:
not sure can it support multiple labels give it has "BoutiquesBidsSingleSubjectMaker":